tools/__tasks__/compile/fonts/index.config.js

```diff
@@ -1,435 +1,507 @@
             },
         ],
     },
     // *** Guardian Text Egyptian - woff2 - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebHinted.woff2.json',
-        'font-family': '"Guardian Text Egyptian Web"', // TODO: Rename to Guardian Text Egyptian?
+        dest: 'GuardianTextEgyptianHinted.woff2.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff2'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff2',
             },
             {
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff2'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff2',
             },
             {
                 'font-weight': '700',
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff2'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff2',
             },
         ],
     },
     // *** Guardian Text Egyptian - woff - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebHinted.woff.json',
-        'font-family': '"Guardian Text Egyptian Web"', // TODO: Rename to Guardian Text Egyptian?
+        dest: 'GuardianTextEgyptianHinted.woff.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff',
             },
             {
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff',
             },
             {
                 'font-weight': '700',
                 'font-style': 'italic',
-                src: 'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff'
+                src:
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff',
             },
         ],
     },
-    // *** Guardian Text Egyptian - ttf - Hinted ***    
+    // *** Guardian Text Egyptian - ttf - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebHinted.ttf.json',
-        'font-family': '"Guardian Text Egyptian Web"', // TODO: Rename to Guardian Text Egyptian?
+        dest: 'GuardianTextEgyptianHinted.ttf.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
                 'font-weight': '300',
```

Comment:
Regular weight should always be 400, I think. 300 is for Light. Also weight numbers are missing for Regular weights above for all flavours but ttf hinted.